README.md

```diff
@@ -1,9 +1,53 @@
-
 # Telegraf
 
-![tiger](assets/TelegrafTiger.png "tiger")
-
-[![Contribute](https://img.shields.io/badge/Contribute%20To%20Telegraf-orange.svg?logo=influx&style=for-the-badge)](https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md) [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=for-the-badge)](https://www.influxdata.com/slack) [![Circle CI](https://circleci.com/gh/influxdata/telegraf.svg?style=svg)](https://circleci.com/gh/influxdata/telegraf) [![GoDoc](https://godoc.org/github.com/influxdata/telegraf?status.svg)](https://godoc.org/github.com/influxdata/telegraf) [![Docker pulls](https://img.shields.io/docker/pulls/library/telegraf.svg)](https://hub.docker.com/_/telegraf/) [![Go Report Card](https://goreportcard.com/badge/github.com/influxdata/telegraf)](https://goreportcard.com/report/github.com/influxdata/telegraf)
+<div align="center">
+  <img width="600" src="assets/telegraf-logo.png" alt="Telegraf Logo">
+</div>
+
+
+
+<p align="center">
+  <a href="https://github.com/influxdata/telegraf/blob/master/CONTRIBUTING.md">
+  <img alt="Contribute" src="https://img.shields.io/badge/Contribute%20To%20Telegraf-orange.svg?logo=influx&style=for-the-badge" />
+  </a>
+  
+  <a href="https://www.influxdata.com/slack">
+  <img alt="Slack Status" src="https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=for-the-badge" />
+  </a>
+    
+  <a href="https://circleci.com/gh/influxdata/telegraf">
+  <img alt="Circle CI" src="https://circleci.com/gh/influxdata/telegraf.svg?style=svg" />
+  </a>
+  
+  <a href="https://hub.docker.com/_/telegraf/">
+  <img alt="Docker Pulls" src="https://img.shields.io/docker/pulls/library/telegraf.svg" />
+  </a>
+    
+  <a href="https://godoc.org/github.com/influxdata/telegraf">
+  <img alt="GoDoc" src="https://godoc.org/github.com/influxdata/telegraf?status.svg" />
+  </a>
+  
+  <a href="https://github.com/influxdata/telegraf/blob/master/LICENSE">
+  <img alt="License" src="https://img.shields.io/github/license/influxdata/influxdb" />
+  </a>
```

Comment:
While we are at it, can we change all shields to be in the same theme? (its possibly by using shields.io for all)